docs/maya/muscle.md

```diff
@@ -114,22 +116,38 @@ In order to provide more artistic control, some key parameters of the muscle sol
 
 | Name | Default | Description |
 | :--- | :------ | :---------- |
-| **Attachment Constraints**       | 0.0             | Weight to indicate the influence of each attachment at each vertex of the muscle. |
-| **Tendons**                      | 0.0             | Floating values to indicate the source of the muscle fibers. The solver will use that information to make an estimation of the fiber direction at each vertex. It is recommended to set a value of 1.0 wherever the tendinous tissue would be in an anatomically realistic muscle and a value of 0.0 in the rest of the mesh. |
-| **Fibers**                       | {0.0, 0.0, 0.0} | The deformer estimates the fiber directions at each vertex based on the tendon weights. In case that the estimated fibers do not fit well to the desired directions, the paint tool can be used to comb the fibers manually. The fibers can be displayed using the *Muscle Fibers* option in the [debugger](#debugger). |
+| **Attachments To Transform**     | 0.0             | Weight to indicate the influence of each transform attachment at each vertex of the muscle. |
+| **Attachments To Geometry**      | 0.0             | Weight to indicate the influence of each geometry attachment at each vertex of the muscle. |
```

Comment:
This is just a thought. It might be worth to mention when an attribute is multi-influence or not

Reply:
Should we do it in the description or adding another column?

Reply:
Maybe to not add too much complexity in the Description instead of `Weight..` we could say `Multi-influence weight...`. Do you agree!

Reply:
Love it!